packages/codecs/src/anchor.ts

```diff
@@ -12,43 +13,46 @@ export enum RequestStatusName {
   REPLACED = 'REPLACED',
 }
 
-export const CommitPresentation = sparse(
+/**
+ * Part of CAS response that sends AnchorCommit content. Effectively a historical artefact.
+ */
+export const AnchorCommitPresentation = sparse(
   {
-    content: optional(
-      sparse({ path: optional(string), prev: string, proof: optional(string) }, 'content')
-    ),
     cid: string.pipe(cidAsString),
   },
-  'CommitPresentation'
+  'AnchorCommitPresentation'
 )
-export type CommitPresentation = TypeOf<typeof CommitPresentation>
+export type AnchorCommitPresentation = TypeOf<typeof AnchorCommitPresentation>
 
 export const NotCompleteStatusName = union([
-  literal(RequestStatusName.PENDING),
-  literal(RequestStatusName.PROCESSING),
-  literal(RequestStatusName.FAILED),
-  literal(RequestStatusName.READY),
-  literal(RequestStatusName.REPLACED),
+  literal(AnchorRequestStatusName.PENDING),
+  literal(AnchorRequestStatusName.PROCESSING),
+  literal(AnchorRequestStatusName.FAILED),
+  literal(AnchorRequestStatusName.READY),
+  literal(AnchorRequestStatusName.REPLACED),
 ])
 export type NotCompleteStatusName = TypeOf<typeof NotCompleteStatusName>
 
 export const NotCompleteCASResponse = sparse(
   {
+    id: string,
```

Comment:
should this be `cidAsString`?

Reply:
It is request id, which is uuid string.

Reply:
oh whoops, good call.  I was thinking it was the `id` field of the AnchorCommit, which is the genesis CID.